firstvoices/backend/search/utils/query_builder_utils.py

```diff
@@ -26,7 +26,7 @@ class SearchDomains(Enum):
 
 def get_indices(types):
     """
-    Returns list of indices to go through depending on the docType
+    Returns list of documents to go through depending on the docType
```

Comment:
This looks like it return indices, not documents

Reply:
Yes sorry, I think a refactor caught a couple of comments